projects/storefrontapp-e2e-cypress/cypress/integration/regression/site-context/language/language-checkout-page.e2e-spec.ts

```diff
@@ -24,16 +26,21 @@ describe('Language switch - checkout page', () => {
 
   describe('populate cart, history, quantity', () => {
     it('should have basic data', () => {
-      siteContextSelector.doPlaceOrder();
       manipulateCartQuantity();
     });
   });
 
   describe('checkout page', () => {
     it('should change language in the shipping address url', () => {
       // page being already tested in language-address-book
+      cy.route(
```

Comment:
make sense. I questioned this before when I was making and talking to the QA, and I am guessing this helped with the flakiness, so I don't mind having this as you said also, it is already being tested.